src/uu/uniq/src/uniq.rs

```diff
@@ -273,7 +315,7 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
             || matches.contains_id(options::GROUP),
         delimiters: get_delimiter(&matches),
         show_counts: matches.get_flag(options::COUNT),
-        skip_fields: opt_parsed(options::SKIP_FIELDS, &matches)?,
+        skip_fields: skip_fields_modern.or(skip_fields_old),
```

Comment:
Maybe we should give an error if both are given? I imagine that it's weird if modern takes precedence over the old syntax even if the old syntax is given after the modern one.

Reply:
since SKIP_FIELDS has `overrides_with_all(OBSOLETE_SKIP_FIELDS_DIGITS)` the one used later overrides the one used earlier, so the modern syntax wouldn't take precedence.

since they are guaranteed not to be both `Some()` i thought `or` was a nice way to put them in a single variable

Reply:
Oh gotcha I missed that :) Excellent